cmd/execute.go

```diff
@@ -186,13 +188,30 @@ func watchEthStorage(w watcher.IStorageWatcher, wg *syn.WaitGroup) {
 	defer wg.Done()
 	// Execute over the StorageTransformerInitializer set using the storage watcher
 	LogWithCommand.Info("executing storage transformers")
-	ticker := time.NewTicker(pollingInterval)
-	defer ticker.Stop()
-	for range ticker.C {
-		errs := make(chan error)
-		diffs := make(chan storageUtils.StorageDiff)
-		w.Execute(diffs, errs, queueRecheckInterval)
+	on := viper.GetBool("storageBackFill.on")
+	if on {
+		backFillStorage(w)
+	}
+	w.Execute(queueRecheckInterval, on)
+}
+
+func backFillStorage(w watcher.IStorageWatcher) {
+	// configure archival rpc client
+	// move this all into the storage watcher?
+	archivalRPCPath := viper.GetString("storageBackFill.rpcPath")
```

Comment:
It's not the same IPC path because this process connects to the http server whereas the subscription process connects to the ws server. Only time they would be same is if they are both using the ipc endpoint and I don't think that is a safe assumption to make.

Reply:
This makes me realize the storage transformers are not being configured with the ws endpoint, if they are using the standard ipc path.